index.html

```diff
@@ -12,7 +12,7 @@ <h1></h1>
     </header>
 
     <main class="header__page-title">
-        Hello
+        This is Mikael speaking!
```

Comment:
Actually I dont like this on this page, please revert

Reply:
Alright, I fixed it